src/api/products/products.service.js

```diff
@@ -1,4 +1,7 @@
 import * as productsRepository from './products.repository.js';
+import * as orderProductsRepository from '../orderProducts/orderProducts.repository.js';
+import * as ordersService from '../orders/orders.service.js';
+import * as genresGameTitlesRepository from '../genres_gameTitles/genres_gameTitles.service.js';
```

Comment:
Either the reference name is wrong or the path is wrong here...

Reply:
resolved